ruma-api/tests/ui/02-invalid-path.rs

```diff
@@ -0,0 +1,21 @@
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Does something.",
+        method: GET,
+        name: "newtype_body_endpoint",
+        path: "/_matrix/some/query/map/\u{ffff}",
```

Comment:
Maybe we could use an example that doesn't require `\u{}`? Something simpler like µ / ° / § / € (al of which can easily be typed on a German keyboard, not sure about others).